docs/_releases/v0.6.0-rc.5t.md

```diff
@@ -74,7 +74,7 @@ language: en
 - [Bug] Show static board results @theBeginner86 (#5438)
 - Bump @fortawesome/react-fontawesome from 0.1.16 to 0.1.18 in /ui @dependabot (#5507)
 - [UI] Select Pattern Component dropdown opening downwards now @Harshit-Dandriyal (#5446)
-- [UI]: Display filename after upload. @SAEb-ai (#5501)
+- [UI] : Display filename after upload. @SAEb-ai (#5501)
```

Comment:
@FilledEther20, thank you.

The whitespace between the square bracket and the colon is improper. Will you identify and use another way to solve this issue, excluding insertion of this whitespace?

e.g. use of a `\`
```
[UI\]: some text here
```